inc/command/class-plugin.php

```diff
@@ -74,20 +101,26 @@ public function install_files() {
 		echo(
 			"\n" .
 			'The file .travis.yml does not match that required by Altis.' . "\n" .
-			'See the file at: ' . $source . '/travis/project.yml' . "\n" .
+			'See the file at: ' . esc_textarea( $source ) . '/travis/project.yml' . "\n" .
```

Comment:
I don't think this code runs in a WordPress context. It runs as part of composer. So you won't have access to WordPress functions. 
Also this would be the wrong escaping function anyway.

Reply:
@mikelittle Thanks for pointing it out, I was thinking this is in WP-CLI context. I have removed it.